### PR TITLE
Update concept.md

### PR DESCRIPTION
**Pull request by anadvu**
Just grammar tweaks

### CODE REVIEW

File: docs/concept.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ Pinta uses layers to create a composite image. Think of layers as a stack of tra
 
 ![Stacked Layer](img/stacked.png)
 
-You can draw on each paper but still see the content of the other sheets through the transparent areas. Intermediate users often deal with images containing many layers. Layers do not need to be opaque and they do not cover the entire extent of an image. So when you look at an image's display, you may see more than just the top layer: you may see elements of many layers. 
+You can draw on each paper but still see the content of the other sheets through the transparent areas. Intermediate users often deal with images containing many layers. Layers do not need to be opaque and they do not cover the entirity of an image. So when you look at an image's display, you may see more than just the top layer: you may see elements of many layers.
```
Comment (jeneira94) [inline docs/concept.md]: I think you spelled **entirety** wrong....

**Review by jeneira94 — CHANGES_REQUESTED** [review]
I can approve once line 17 is fixed.

File: docs/concept.md (hunk 1 above)
Comment (anadvu) [reply docs/concept.md]: You're right 🤦 no more typing past 1 am haha

Comment (jeneira94) [reply docs/concept.md]: haha no worries

**Pull request merged**